Bug: No module named 'tiktoken_async'

### 操作系统

Linux

### Python 版本

3.11.2

### NoneBot 版本

1.2.6

### Bilichat 版本

4.12.0

### 描述问题

nonebot启动失败，无法加载 `nonebot_plugin_bilichat`

### 插件的配置项

_No response_

### 截图或日志


![image](https://github.com/Well2333/nonebot-plugin-bilichat/assets/958456/32a48a71-b0a2-4524-a7d7-dd4dc7ded162)

```
12-06 10:09:20 [ERROR] nonebot | Failed to import "nonebot_plugin_bilichat"
Traceback (most recent call last):
  File "<string>", line 14, in <module>
  File "/root/qqbot/.venv/lib/python3.11/site-packages/nonebot/plugin/load.py", line 130, in load_from_toml
    return load_all_plugins(plugins, plugin_dirs)
  File "/root/qqbot/.venv/lib/python3.11/site-packages/nonebot/plugin/load.py", line 64, in load_all_plugins
    return manager.load_all_plugins()
  File "/root/qqbot/.venv/lib/python3.11/site-packages/nonebot/plugin/manager.py", line 176, in load_all_plugins
    return set(
  File "/root/qqbot/.venv/lib/python3.11/site-packages/nonebot/plugin/manager.py", line 177, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.available_plugins))
> File "/root/qqbot/.venv/lib/python3.11/site-packages/nonebot/plugin/manager.py", line 142, in load_plugin
    module = importlib.import_module(name)
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1206, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "/root/qqbot/.venv/lib/python3.11/site-packages/nonebot/plugin/manager.py", line 242, in exec_module
    super().exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/qqbot/.venv/lib/python3.11/site-packages/nonebot_plugin_bilichat/__init__.py", line 67, in <module>
    from . import adapters, commands, subscribe  # noqa: F401, E402
  File "/root/qqbot/.venv/lib/python3.11/site-packages/nonebot_plugin_bilichat/subscribe/__init__.py", line 9, in <module>
    from .dynamic import fetch_dynamics_grpc, fetch_dynamics_rest
  File "/root/qqbot/.venv/lib/python3.11/site-packages/nonebot_plugin_bilichat/subscribe/dynamic.py", line 15, in <module>
    from ..adapters.base_content_parsing import check_cd
  File "/root/qqbot/.venv/lib/python3.11/site-packages/nonebot_plugin_bilichat/adapters/base_content_parsing.py", line 13, in <module>
    from ..summary.text_to_image import t2i
  File "/root/qqbot/.venv/lib/python3.11/site-packages/nonebot_plugin_bilichat/summary/__init__.py", line 6, in <module>
    from .openai_summarise import openai_summarization
  File "/root/qqbot/.venv/lib/python3.11/site-packages/nonebot_plugin_bilichat/summary/openai_summarise.py", line 9, in <module>
    from .openai import get_small_size_transcripts, get_summarise_prompt, openai_req
  File "/root/qqbot/.venv/lib/python3.11/site-packages/nonebot_plugin_bilichat/summary/openai.py", line 7, in <module>
    import tiktoken_async
ModuleNotFoundError: No module named 'tiktoken_async'
```

--- a/nonebot_plugin_bilichat/adapters/base_content_parsing.py
+++ b/nonebot_plugin_bilichat/adapters/base_content_parsing.py
@@ -9,8 +9,8 @@ from nonebot.typing import T_State
 
 from ..config import plugin_config
 from ..content import Column, Dynamic, Video
+from ..lib.text_to_image import t2i
 from ..model.exception import AbortError
-from ..summary.text_to_image import t2i
 
 if plugin_config.bilichat_openai_token:
     ENABLE_SUMMARY = True

--- a/nonebot_plugin_bilichat/lib/text_to_image.py
+++ b/nonebot_plugin_bilichat/lib/text_to_image.py
@@ -9,8 +9,8 @@ from dynrender_skia.DynText import BiliText
 from nonebot.log import logger
 
 from ..config import plugin_config
-from ..lib.fonts_provider import get_font_sync
-from ..lib.store import cache_dir
+from .fonts_provider import get_font_sync
+from .store import cache_dir
 
 data_path = cache_dir.joinpath("render")
 data_path.mkdir(parents=True, exist_ok=True)

--- a/nonebot_plugin_bilichat/summary/openai_summarise.py
+++ b/nonebot_plugin_bilichat/summary/openai_summarise.py
@@ -4,10 +4,10 @@ from nonebot.log import logger
 
 from ..config import plugin_config
 from ..lib.cache import BaseCache
+from ..lib.text_to_image import t2i
 from ..model.exception import AbortError
 from ..optional import capture_exception  # type: ignore
 from .openai import get_small_size_transcripts, get_summarise_prompt, openai_req
-from .text_to_image import t2i
 
 
 async def subtitle_summarise(title: str, sub: List[str]):

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,7 +1,7 @@
 [project]
 name = "nonebot-plugin-bilichat"
 
-version = "4.12.0"
+version = "4.12.1"
 
 description = "多种B站链接解析，视频词云，AI总结，你想要的都在 bilichat"
 authors = [

